static/src/javascripts/projects/commercial/modules/dfp/queue-advert.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+
+import dfpEnv from 'commercial/modules/dfp/dfp-env';
+
+const queueAdvert = (advert: Object) => {
+    dfpEnv.advertsToLoad.push(advert);
+    // Add to the array of ads to be refreshed (when the breakpoint changes)
+    // only if its `data-refresh` attribute isn't set to false.
+    if (advert.node.getAttribute('data-refresh') !== 'false') {
+        dfpEnv.advertsToRefresh.push(advert);
+    }
+};
+
+export default queueAdvert;
```

Comment:
I would certainly like to use named exports. But if we do it after `commercial` is fully migrated, it avoids having to do `module.something` in legacy scripts, only to update them to `something` a few days later. Is that ok?